### DISCUSSION TITLE
What is the purpose of the language parameter in `Pre-Recorded`

### DISCUSSION THREAD

**Discussion by last-misaka** [deepgram/community#690] [General help] [answered]
"Hello! I've been exploring the Deepgram API for Pre-Recorded audio transcription and have read through the language support documentation ([developers.deepgram.com/docs/language](https://developers.deepgram.com/docs/language)). However, I'm still unclear on a couple of points and would appreciate some clarification:

- Does the recognition accuracy significantly decrease if an audio file's spoken language does not match the one specified in the language parameter?

Thank you in advance for any insights or guidance!"

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@lzeiw  

The[ language parameter ](https://developers.deepgram.com/docs/language)  specifies the primary spoken language you wish to define of the spoken audio when you make a request to our API.

It is best practice to specify the actual language used in the audio file for best results.

Though you can also use our [detect language](https://developers.deepgram.com/docs/language-detection) feature which does support a number of languages. If you do this you don't need to provide the specific language. Just double check that we support language you need in our Language Detection Docs as not all languages ares supported currently.

**Reply by last-misaka** [reply]
much appreciated !
